src/interfaces/fees.ts

```diff
@@ -6,18 +6,26 @@ import { PROTOCOL_FEES } from "../services/subgraph/queries";
 import { FeesResponse } from "../services/subgraph/responses";
 import { Usdc } from "../types";
 
+const BigZero = new BigNumber(0);
```

Comment:
We've introduced a helper to work with big numbers similar to what we have in the frontend
```suggestion
const BigZero = toBN(0);
```

Reply:
Also, as mentioned below https://github.com/yearn/yearn-sdk/pull/285/files#r859174071, I don't think this is needed